internal/installation/paths.go

```diff
@@ -9,16 +10,23 @@ import (
 	"github.com/ActiveState/cli/internal/condition"
 	"github.com/ActiveState/cli/internal/constants"
 	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/exeutils"
 	"github.com/ActiveState/cli/internal/fileutils"
 )
 
-// CfgInstallPath is the configuration key for the path where the State Tool is installed
-const CfgInstallPath = "installation_path"
+const (
+	// CfgInstallPath is the configuration key for the path where the State Tool is installed
+	CfgInstallPath = "installation_path"
 
-// CfgTransitionalStateToolPath is the configuration key for the path where a transitional State Tool might still be stored
-const CfgTransitionalStateToolPath = "transitional_installation_path"
+	// CfgTransitionalStateToolPath is the configuration key for the path where a transitional State Tool might still be stored
+	CfgTransitionalStateToolPath = "transitional_installation_path"
 
-const BinDirName = "bin"
+	BinDirName = "bin"
+
+	InstallDirMarker = ".install_root"
```

Comment:
We should probably name this something more activestate specific. eg. `.state_install_root`.